haiku-apps/tolmach/tolmach-1.1.0.recipe

```diff
@@ -1,14 +1,14 @@
 SUMMARY="A dictionary for Haiku"
-DESCRIPTION="
-This program is a BeOS port of KDictionary translation program for Linux by \
-Ivan V. Murasko. It follows the main concept of original linux program but \
-mainly is a real BeOS program.
-"
+DESCRIPTION="This program is a BeOS port of KDictionary translation \
+program for Linux by Ivan V. Murasko. It follows the main concept of \
+original linux program but mainly is a real BeOS program."
+HOMEPAGE="https://gitorious.org/desknotes/tolmach"
 COPYRIGHT="1999 Ivan V. Murasko"
 LICENSE="GNU GPL v2"
-HOMEPAGE="https://gitorious.org/desknotes/tolmach"
-SOURCE_URI="git+http://github.com/haikuarchives/tolmach.git#5a67e9d38292d5dd482e116e8ee7aa605dee5398"
 REVISION="1"
+SOURCE_URI="https://github.com/haikuarchives/tolmach/archive/5a67e9d38292d5dd482e116e8ee7aa605dee5398.tar.gz"
+CHECKSUM_SHA256="44da1dd27ff36465be712fc21ace226e00cdd1acd314074ae473ab31642a3cff"
+SOURCE_DIR="Tolmach-5a67e9d38292d5dd482e116e8ee7aa605dee5398"
 
 ARCHITECTURES="x86_gcc2 x86 ?x86_64"
 
```

Comment:
No empty lines between PROVIDES and REQUIRES and BUILD_REQUIRES and BUILD_PREREQUIRES.